examples/run_script_with_local_browser/run_script_with_local_browser.py

```diff
@@ -0,0 +1,77 @@
+"""This example demonstrates how to run the script in an existing local browser with AgentQL."""
+
+from agentql.ext.playwright.sync_api import Page
+from playwright.sync_api import sync_playwright
+
+# The URL to connect to your local existing browser.
+WEBSOCKET_URL = "YOUR_WEBSOCKET_URL"
+
+URL = "https://scrapeme.live/shop"
+
+QUERY = """
+{
+    tours[] {
+        title
+        price
+        length
+    }
+}
+"""
+
+QUERY_FOR_SCRAPE_ME_1 = """
```

Comment:
I suggest to use get_by_prompt if we want to get 1 element (maybe for the first one), and/or add another element to the query (the second query) to use it as query_elements